examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -333,6 +299,8 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
       countmapMapResult: [],
       countmapPartialMapResult: [],
       commonTimeSeriesResult: [],
+      cachedTimeSeriesResult: [],
```

Comment:
Put a comment about these two variables. What's their functionality? Other developers might be confused. Can you come up with a more clear name?